src/Routes/ListRoutesCommandFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mezzio\Tooling\Routes;
+
+use Mezzio\Router\RouteCollector;
+use Psr\Container\ContainerInterface;
+
+class ListRoutesCommandFactory
+{
+    public function __invoke(ContainerInterface $container): ListRoutesCommand
+    {
+        $routesCollector = $container->get(RouteCollector::class);
```

Comment:
If you need the container to be typed, a stub may be added to this lib.

Reply:
Not following, sorry.

Reply:
Psalm does not automagically understand that if you fetch something from the container what the return value is.
That's why psalm tells you that "mixed" is being assigned to the `$routesCollector` variable.

I usually avoid this by using this psalm plugin:

https://github.com/Lctrs/psalm-psr-container-plugin